src/app/util/attribute-storage.cpp

```diff
@@ -390,7 +390,9 @@ CHIP_ERROR emberAfSetDynamicEndpointWithEpUniqueId(uint16_t index, EndpointId id
     }
 
     // Now enable the endpoint.
-    emberAfEndpointEnableDisable(id, true);
+    VerifyOrReturnError(chip::DeviceLayer::SystemLayer().ScheduleLambda([id]() { emberAfEndpointEnableDisable(id, true); }) ==
```

Comment:
same here: update the comment to say why schedulelambda is important instead of direct inline.